test/integration/users_login_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.